src/cli/cli.ts

```diff
@@ -1,45 +1,75 @@
-import { getLanguages, LanguageCodes, Sources } from '..';
+import { getLanguages,  Sources } from '..';
 import { fileTranslator, getFileFromPath } from '../core/json_file';
 import {
-  commands,
   error,
   info,
-  language_choices,
   messages,
   success,
   warn,
 } from '../utils/console';
 import loading from 'loading-cli';
-import { getCodeFromLanguage, translationStatistic } from '../utils/micro';
+import { capitalize, current_version, getCodeFromLanguage, translationStatistic } from '../utils/micro';
 import { readProxyFile } from '../core/proxy_file';
-var inquirer = require('inquirer');
+import { Command } from "commander";
+import { promptFrom, promptTo, promptTranslator } from '../utils/prompt';
+var figlet = require("figlet");
+
+const program = new Command();
 
 export async function initializeCli() {
   global.totalTranslation = 0;
   global.totalTranslated = 0;
   global.proxyIndex = 0;
   global.proxyList = [];
+  program
+    .version(current_version)
+    .usage(messages.cli.usage)
+    .description(`${messages.cli.usageWithProxy}\n${messages.cli.usageByOps}\n\n\n${messages.cli.description}`)
+    .option("-T, --translator <TranslationService>", messages.cli.translator, false)
+    .option("-f, --from  <Language>", messages.cli.from, false)
+    .option("-t, --to <Languages>", messages.cli.to, false)
```

Comment:
`commander` also have a syntax that supports variadic options by spreading (`<Languages...>`), I believe that it would be better than manually parse user input, because I personally prefer to use power that 3rd party tools provide us ^_^

Reply:
When I find free time, I will consider all your suggestions regarding "withdrawal request." Because I rushed a little in implementation. I think you found some strange things 👽 in the code 😅. I will dedicate myself in the near future, and it will be a good job 👌